test/units/galaxy/test_collection.py

```diff
@@ -40,9 +40,9 @@ def reset_cli_args():
 
 
 @pytest.fixture()
-def collection_input(tmp_path_factory):
+def collection_input(tmp_path_factory, path_sufix='test-ÅÑŚÌβŁÈ Collections Input'):
     ''' Creates a collection skeleton directory for build tests '''
-    test_dir = to_text(tmp_path_factory.mktemp('test-ÅÑŚÌβŁÈ Collections Input'))
+    test_dir = to_text(tmp_path_factory.mktemp(path_sufix))
```

Comment:
```suggestion
@pytest.fixture
def collection_path_suffix(request):
    """Return test collection path suffix or the default."""
    return getattr(request, 'param', 'test-ÅÑŚÌβŁÈ Collections Input')

@pytest.fixture
def collection_input(tmp_path_factory, collection_path_suffix):
    """Create a collection skeleton directory for build tests."""
    test_dir = to_text(tmp_path_factory.mktemp(collection_path_suffix))
```

Reply:
Ill rollback the commit to get these changes ++

Reply:
Sure, you can follow https://hynek.me/til/easier-crediting-contributors-github/ or just use my <username> AT redhat.com.